lib/handlers/get.js

```diff
@@ -76,7 +76,9 @@ function handler (req, res, next) {
     // Till here it must exist
     if (!includeBody) {
       debug('HEAD only')
-      res.sendStatus(200)
+      contentType = mime.contentType(_path.extname(path))
```

Comment:
Not complete, since the current server assumes that extensionless paths are Turtle.

I would suggest to use an instance of LegacyResourceMapper here. It's overkill for now, but will give you the correct answer, and can in the future easily be replaced by ResourceMapper (which has a different MIME type detection system). See #661 and #662.

You can see an example instantiation in `lib/handlers/allow.js`.